manifests/init.pp

```diff
@@ -96,7 +96,7 @@
   $external_url          = undef,) inherits zendserver::params {
   validate_bool($manage_repos)
   validate_re($webserver, ['\Aapache|nginx\Z',], 'Only apache or nginx are supported.')
-  validate_re($phpversion, ['\A5.3|5.4|5.5|5.6|7.0|7.1\Z',], 'Only versions 5.4 to 7.1 are supported.')
+  validate_re($phpversion, ['\A5.3|5.4|5.5|5.6|7.0|7.1|7.2|7.3|7.4\Z',], 'Only versions 5.4 to 7.4 are supported.')
```

Comment:
```suggestion
  validate_re($phpversion, ['\A5.6|7.1|7.2|7.3|7.4\Z',], 'Only versions 5.6 and 7.1 - 7.4 are supported.')
```

These are the only PHP versions supported by current and LTS versions of Zend Server.